.check.exs

```diff
@@ -1,7 +1,7 @@
 [
   tools: [
     {:compiler, env: %{"MIX_ENV" => "test"}},
-    {:dialyzer, false},
+    {:dialyzer, env: %{"MIX_ENV" => "test"}},
```

Comment:
So as I've already pointed out in previous review (https://github.com/karolsluszniak/ex_check/pull/30#discussion_r608797368), adding dialyzer to this project makes no sense as it's not needed in any way and will introduce slowdown for developers and on CI. Adding it to the project is a big change & a lot of code changes below that you didn't explain in this PR.

Reply:
Well, I'm not so sure that this will introduce a slowdown - a check step with dialyzer enabled takes just 10 seconds longer than without dialyzer, in case than dependencies and plts files are cached and reused in the next step. But OK, I've removed this change from PR.